modules/aws-sso/providers.root.tf

```diff
@@ -0,0 +1,15 @@
+provider "aws" {
+  alias  = "root"
+  region = var.region
+
+  # Profile is deprecated in favor of terraform_role_arn. When profiles are not in use, terraform_profile_name is null.
+  profile = module.iam_roles.terraform_profile_name
+
+  dynamic "assume_role" {
+    # module.iam_roles.terraform_role_arn may be null, in which case do not assume a role.
+    for_each = compact([module.iam_roles.terraform_role_arn])
+    content {
+      role_arn = module.iam_roles.terraform_role_arn
+    }
+  }
```

Comment:
do we need all of this? The old root.aws provider only had alias and region
```suggestion
```

Reply:
https://cloudposse.slack.com/archives/CA4TC65HS/p1688064489135749